scripts/autopilot_prompts/fix_round.md

```
@@ -0,0 +1,97 @@
+     Tests and unrelated files are off-limits.
+   - Prefer the suggestion in CR's `<details><summary>建议修复</summary>`
+     block when present — that's an authoritative hint.
+
+4. **Run the validation steps** before committing:
+   - `python -c "import ast; [ast.parse(open(f).read()) for f in changed_py_files]"`
+   - `python <each_changed_script> --help > /dev/null`
+   - `python -c "from lxml import etree; [etree.parse(p) for p in changed_svg_files]"`
+   - `python -c "import yaml; [yaml.safe_load(open(p)) for p in changed_yaml_files]"`
+
+5. **Commit** with a bilingual message in this exact shape:
+   ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)